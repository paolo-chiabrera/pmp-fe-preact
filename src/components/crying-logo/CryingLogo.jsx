import { h } from 'preact';

export default function CryingLogo() {
  return (
    <div class="crying-logo">
      <div class="crying crying-1" />
      <div class="crying crying-2 to-shift" />
      <div class="crying crying-3" />
      <div class="crying crying-4 to-shift" />
      <div class="crying crying-5" />
    </div>
  );
}
