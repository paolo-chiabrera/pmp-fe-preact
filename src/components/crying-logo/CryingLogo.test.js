import { h } from 'preact';

import render from 'preact-render-to-string';

import CryingLogo from './CryingLogo.jsx';

describe('Given a CryingLogo component', () => {
  it('should render correctly', () => {
    const tree = render(<CryingLogo />);

    expect(tree).toMatchSnapshot();
  });
});
