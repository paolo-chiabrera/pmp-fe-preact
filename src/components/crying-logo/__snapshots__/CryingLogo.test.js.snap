// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a CryingLogo component should render correctly 1`] = `"<div class=\\"crying-logo\\"><div class=\\"crying crying-1\\"></div><div class=\\"crying crying-2 to-shift\\"></div><div class=\\"crying crying-3\\"></div><div class=\\"crying crying-4 to-shift\\"></div><div class=\\"crying crying-5\\"></div></div>"`;
